### DISCUSSION TITLE
Adding a waypoint in car navigation mode annoying

### DISCUSSION THREAD

**Discussion by Angelman81** [organicmaps/organicmaps#8353] [Ideas]
I tried to use the road navigation for cars (on my motorbike, but that doesn't matter). When you want to take a certain route via additional waypoints as "shaping points", after the long press on a street I need several attemps until the waypoint cursor is on the street. Mostly you unintentionally select the green fields left and right of the street.   Then you are not reaching the waypoint while passing by driving on the street and the navigation is messed up. 
I have small fingers, but I always have to zoom in a lot before there's a chance to set the marker directly onto a road.
.
Wouldn't it be a good idea to let the cursor jump to the closest public road nearby the finger press area while in car navigation mode? 
It's solved like this in other apps like Kurviger.

Feel free to delete this idea if someone else has already mentioned it, I can't find something like this right now :)

**Comment by biodranik** [comment]
This is a good idea, likely not only for the car nav mode but for any other mode too. Especially in the context of #7876 and to display more useful information about any road.
